node/overseer/overseer-gen/proc-macro/src/impl_builder.rs

```diff
@@ -385,6 +406,11 @@ pub(crate) fn impl_builder(info: &OverseerInfo) -> proc_macro2::TokenStream {
 				#baggage_name: #baggage_passthrough_state_generics,
 			)*
 			spawner: InitStateSpawner,
+			// user provided runtime overrides,
+			// if `None`, the `overlord(message_capacity=123,..)` is used
+			// or the default value.
+			channel_capacity: Option<usize>,
+			signal_capacity: Option<usize>,
```

Comment:
are you planning to add a separate flag for signal capacity? otherwise, why is it configurable?

Reply:
It'd make sense to make the signal capacity the same size as the defined one. I didn't expose it since I don't think it's reasonable, but I wanted to spare anyone having to extend the builder pattern.

Reply:
I was more leaning towards YAGNI :P

Reply:
I tend to violate this, when it comes to proc-macros. I seem to be one of the few that enjoys them :upside_down_face: - so I tend to prioritize minimizing my interruptions and the exposure to proc-macros for those that don't enjoy em.